README.md

```diff
@@ -122,6 +122,11 @@ We're currently doing up to two **new** events per region per quarter to help en
     + Expected size: <200
     + Quarter: Spring 2019
     + Additional info: 
+- Chicago, USA
```

Comment:
can you pls also update the number of slots filled above?

Reply:
Yep, I just did!